### ISSUE TITLE
Control uses of rand() in test scripts

### ISSUE THREAD

**Issue by plbossart** [thesofproject/sof-test#301]
The test scripts use random number generator to e.g. change the sleep times.

This is wonderful for stress tests, but a really bad idea for CI and daily tests, since by definition these tests cannot be reproduced UNLESS the seed is known.

We should add one option to tests that use random numbers, so that by default we shall use the same seed - 42
https://en.wikipedia.org/wiki/Phrases_from_The_Hitchhiker%27s_Guide_to_the_Galaxy#Answer_to_the_Ultimate_Question_of_Life,_the_Universe,_and_Everything_(42)

When the user wishes more randomness, the seed shall be saved in the logs so that we can reproduce failures.

This can be done in bash with the RANDOM variable

Normal case:
````bash
$ T=42
$ RANDOM=$T; echo $RANDOM; echo $RANDOM; echo $RANDOM
17766
11151
23481
$ RANDOM=$T; echo $RANDOM; echo $RANDOM; echo $RANDOM
17766
11151
23481
````

stress-test:

````bash
$ T=$(date +%s)
$ echo $T
1596129567
$ RANDOM=$T; echo $RANDOM; echo $RANDOM;
19204
11943
$ RANDOM=$T; echo $RANDOM; echo $RANDOM;
19204
11943
$ RANDOM=$T; echo $RANDOM; echo $RANDOM;
19204
11943
````

**Comment by marc-hb** [comment]
As suggested by @xiulipan : the best option is a command flag that lets the caller choose between constant seed versus random seed.

My suggestions:
- The test should always print the seed used in any case so it can be re-used.
- clearly label CI results using a random seed, e.g. "stress test" not to surprise users.

**Comment by xiulipan** [comment]
thanks @marc-hb for the suggestion. I will take them with the new feature implement.
Another feature: 

- use a parameter to accept seed, used for reproduce issue from stress test

**Comment by marc-hb** [comment]
> use a parameter to accept seed, used for reproduce issue from stress test

Yes that's what I meant by "... so it can be re-used". Rephrasing the previous comment.
